VMBackup/main/handle.py

```diff
@@ -165,24 +165,24 @@ def snapshot():
         backup_logger.log(errMsg, False, 'Error')
     snapshot_done = True
 def daemon():
-    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser
+    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser,snapshot_done
```

Comment:
spaces between those would be more useful. 

have you tried running pep8 on this file?